apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -193,6 +194,7 @@ class Item extends Component {
             className="pop-up-menu-item"
             href={href}
             style={{...textStyle, ...areaStyle}}
+            target="_blank"
```

Comment:
we use pop up menu in other places, do we always want this to be true?

Reply:
Good point.  I made a `openInNewTab` prop that can be passed to PopUp Items to conditionally open a new tab for links.